CHANGELOG.md

```diff
@@ -1,16 +1,34 @@
 # GIDS Health Tools Interoperability Changelog
 
+---
+## HTI:core version 2.0
+Document version: 2.0.0
+Date: 30-01-2023
+
+This is a breaking change. The FHIR Task object turns out to be used in completely ways over different contexts. As the
+FHIR Task object plays a central role in HTI, this has become troublesome. The main points are:
+
+ * There is little to none consensus over the meaning of the most used fields (for, owner).
+ * The above outcome has resulted in the observation that the usage of the FHIR task from one domain to another seems impossible.
+ * The standard has changed not only the structure of the reference to the activityDefinition between FHIR R3 and R4, but also the contents of the field (from reference to the value of the uri).
+
+In order to resolve this, the FHIR Task object will be replaced with a set of extra defined fields, except for the Task.for,
+this field will be mapped to the sub field of the JWT token.
+
+Documentation updates:
+ * Issue #18: Request: let go of the nested FHIR task and use own defined fields instead
+
 ---
 ## HTI:core version 1.1
```

Comment:
the following changes are cosmetics only. Perhaps they can be commited to master already which would make the PR easier to read.